docs/docs/state.md

```diff
@@ -4,25 +4,23 @@ title: Handler State
 sidebar_label: Handler State
 ---
 
-As described in ["About Gesture Handlers"](about-handlers.md) section gesture handlers can be treated as ["state machines"](https://en.wikipedia.org/wiki/Finite-state_machine).
-Each handler instance at any given time has an assigned state that can change when new touch events arrive or can be forced by the touch system to change it state in certain circumstances.
+As described in ["About Gesture Handlers"](about-handlers.md), gesture handlers can be treated as ["state machines"](https://en.wikipedia.org/wiki/Finite-state_machine).
+At any given time, each handler instance has an assigned state that can change when new touch events occur or can be forced to change by the touch system in certain circumstances.
 
-There are six possible states for the handler:
+A gesture handler can have six possible states:
```

Comment:
```suggestion
A gesture handler can be in one of the six possible states:
```

There are 6 states overall, but the handler can only be in one (a state machine)

Reply:
excellent